x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +60,24 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().String(FlagNamespaceVersion, "0", "Specify the namespace version (default is 0)")
```

Comment:
why use a string here? if we use a int, would it be easier to convert to uint8?

Reply:
yes, changed to uint8. Thanks